src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import moreIcon from './more.svg';
+import reload from './reload.svg';
+import enter from './enter.svg';
+
+const ToDoListContainer = document.getElementById('list-container');
+const reloadIcon = document.getElementById('reload-img');
+const enterIcon = document.getElementById('enter-icon');
+enterIcon.src = enter;
+reloadIcon.src = reload;
+const todoList = [
+  {
+    id: 1,
+    description: 'Starting project structure',
+    completed: false,
+  },
```

Comment:
- [ ] Dear @AndresEspi, as stated in the project requirement, for Each task we need to use an `index` instead of an `id`. Please update your task list to have `index`, `description` and `completed`.